CoreScriptsRoot/Modules/BackpackScript.lua

```diff
@@ -97,6 +107,8 @@ local OpenInventoryButton = nil
 local CloseInventoryButton = nil
 local InventoryFrame = nil
 local ScrollingFrame = nil
+local ScrollUpInventoryButton = nil
+local ScrollDownInventoryButton = nil
```

Comment:
Outside of the scope of this task, but leaving comment for future reference. Should probably fix these forward declares and create the gui objects at the start of the script. Made it a bit confusing since they do get initialized in the main scope, but functions before they are initialized use them. Wasn't sure at first if they needed to be nil checked or not.